app/design/frontend/rwd/default/template/catalog/product/list.phtml

```diff
@@ -47,7 +48,7 @@
     <?php $_iterator = 0; ?>
     <ol class="products-list" id="products-list">
     <?php foreach ($_productCollection as $_product): ?>
-        <li class="item<?php if( ++$_iterator == sizeof($_productCollection) ): ?> last<?php endif ?>">
+        <li class="item<?php if(++$_iterator === count($_productCollection)): ?> last<?php endif ?>">
```

Comment:
`count($_productCollection)` can be replaced by `$_productCollectionCount` declared above.

Reply:
Thanks, I missed that one.